json-rpc/src/runtime.rs

```diff
@@ -93,6 +102,7 @@ async fn rpc_endpoint(
     data: Value,
     service: JsonRpcService,
     registry: Arc<RpcRegistry>,
+    chain_id: ChainId,
```

Comment:
can we store chain_id in JsonRpcService instead?

Reply:
yeah, good idea, fixed.